src/System.Runtime.Environment/tests/packages.config

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="xunit" version="2.0.0-beta5-build2785" />
```

Comment:
I recommend rebasing this on master to pick up Eric's project.json change, moving these dependencies from packages.config to project.json, and bumping the version number of the packages.